Sources/MapboxNavigation/NavigationMapViewDelegate.swift

```diff
@@ -107,36 +99,4 @@ public extension NavigationMapViewDelegate {
         logUnimplemented(protocolType: NavigationMapViewDelegate.self, level: .debug)
         return .zero
     }
-}
-
-// MARK: - NavigationMapViewCourseTrackingDelegate methods
-
-/**
- The `NavigationMapViewCourseTrackingDelegate` provides methods for responding to the `NavigationMapView` starting or stopping course tracking.
- */
-public protocol NavigationMapViewCourseTrackingDelegate: class, UnimplementedLogging {
```

Comment:
`NavigationMapViewCourseTrackingDelegate` was removed in favor of `Notification`, which is sent whenever `NavigationCameraState` changes. `NavigationCameraState` also provides more information regarding camera state (`.idle`, `.transitionToFollowing`, `.following`, `.transitionToOverview`, `.overview`).

Reply:
Yes, in this case `NavigationMapViewCourseTrackingDelegate` is not enough though, as we need camera state related changes in more than one place.